Makefile

```diff
@@ -8,6 +8,7 @@ install:
 	pip3.6 install -U Cython
 	cd py-libzfs && python3.6 setup.py build && python3.6 setup.py install
 	pkg install -q -y libgit2
+	gzip --keep --force man/iocage.8.gz
```

Comment:
Yeah, we can remove this step, because we create the man page on build time now (see https://github.com/iocage/iocage/pull/267/commits/e31bc08092f74f245d5de3c9e7e9d6762db0acaa). Thanks for reporting!